Client/Help/src/app/components/content-connections-gallery/content-connections-gallery.component.html

```diff
@@ -0,0 +1,10 @@
+<main class="grid grid-cols-2 place-items-center mt-4">
```

Comment:
I think there is room to improve the CSS on this page. On a larger screen, two columns results in a lot of whitespace.

Reply:
I added code to add more columns. But on hindsight, the problem is the app screen size needs to be centralize on larger screens. I didn't add design responsiveness for larger screens. That will be corrected.

Reply:
It has been a painful lesson but I have restyled the desktop design responsiveness.